### ISSUE TITLE
jinja-variable-format false positive of words within quotes.

### ISSUE THREAD

**Issue by jccardonar** [aristanetworks/j2lint#89]
Words with hyphens within a string not touching a quote are raising `jinja-variable-format` false positives:

For instance:

```
{{ "test with spcial-word in the middle" | indent(2, true) }}
```

**Comment by chriscroome** [comment]
I also hit this issue with a Ansible Valkey template containing:

```
pidfile {{ valkey_instance.config.pidfile | default('/run/valkey/valkey-server.pid') }}

logfile {{ valkey_instance.config.logfile | default('/var/log/valkey/valkey-server.log') }} 
```

The above triggered:

```
────────────────────────────── JINJA2 LINT ERRORS ──────────────────────────────
templates/valkey.conf.j2
├── templates/valkey.conf.j2:350 If variable is multi-words, underscore `_` 
│   should be used as a separator: '{{ my_variable_name }}': valkey-server 
│   (jinja-variable-format)
└── templates/valkey.conf.j2:364 If variable is multi-words, underscore `_` 
    should be used as a separator: '{{ my_variable_name }}': valkey-server 
    (jinja-variable-format)

Jinja2 linting finished with 2 error(s) and 0 warning(s)
```

**Comment by cdtomkins** [comment]
+1

```
> cat test.j2 
example{{ ' false-positive ' + entry.thing if entry.thing }}
example{{ " false-positive " + entry.thing if entry.thing }}
example{{ ' works_okay ' + entry.thing if entry.thing }}
example{{ " works_okay " + entry.thing if entry.thing }}
> j2lint test.j2 
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── JINJA2 LINT ERRORS ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
test.j2
├── test.j2:1 If variable is multi-words, underscore `_` should be used as a separator: '{{ my_variable_name }}': false-positive (jinja-variable-format)
└── test.j2:2 If variable is multi-words, underscore `_` should be used as a separator: '{{ my_variable_name }}': false-positive (jinja-variable-format)

Jinja2 linting finished with 2 error(s) and 0 warning(s)
> j2lint --version
Jinja2-Linter Version v1.1.0
```

**Comment by cdtomkins** [comment]
This will be closed by https://github.com/aristanetworks/j2lint/pull/132
